### PR TITLE
Add Submit feedback class

### PR DESCRIPTION
**Pull request by sara-02**
For issue openshiftio/openshift.io#1740
Depends on https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/492 to be merged first, it will fail to build otherwise.

### CODE REVIEW

**Comment by sara-02** [comment]
[test]

File: Dockerfile [hunk 1]
```diff
@@ -35,7 +35,7 @@ RUN pushd /coreapi && \
     # needed for DB migrations
     find coreapi/ -mindepth 1 -maxdepth 1 \( ! -name 'alembic*' -a ! -name hack \) -exec rm -rf {} +
 
-RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@${F8A_WORKER_VERSION}
+RUN pip3 install git+https://github.com/sara-02/fabric8-analytics-worker.git@${F8A_WORKER_VERSION}
```
Comment (miteshvp) [inline Dockerfile]: let's not use forked repo branch here. Please create a PR in worker repo first. and Then use the short commit here

File: bayesian/api_v1.py [hunk 2]
```diff
@@ -952,6 +954,50 @@ def get():
...
+        input_json = request.get_json()
+        if not request.json:
+            raise HTTPError(400, error="Expected JSON request")
+
+        stack_id = input_json.get('stack_id')
+        recommendation_type = input_json.get('recommendation_type')
+        package_name = input_json.get('package_name')
+        feedback_type = input_json.get('feedback_type')
+        ecosystem_name = input_json.get('ecosystem')
+        if not is_valid(stack_id) and \
```
Comment (miteshvp) [inline bayesian/api_v1.py]: or something like
```
condition = [is_valid(stack_id), is_valid(recommendation_type), is_valid(package_name), is_valid(feedback_type), is_valid(ecosystem_name)]
if not all(condition):
    raise ...
```

**Comment by miteshvp** [comment]
@sara-02 what about the integration tests?

**Comment by sara-02** [comment]
@miteshvp I have added them [here](https://github.com/fabric8-analytics/fabric8-analytics-common/pull/459), will wait for @tisnik's comment.

**Comment by miteshvp** [comment]
Ok, in that case, I will merge it.

File: bayesian/utils.py [hunk 3]
```diff
@@ -677,3 +677,8 @@ def _companion_reason(self, manifest_response):
             # Count reason
             pkg["reason"] = count_sentence
         return manifest_response
+
+
+def is_valid(param):
+    """Return true is the param is not a null value."""
+    return param is not None
```
Comment (tisnik) [inline bayesian/utils.py]: __if__ the param

**Comment by miteshvp** [comment]
Revert to see if E2E tests passes.

**Pull request merged**